src/core/components.py

```diff
@@ -496,7 +488,16 @@ def validate_options(self, options_key, options_values):
         return True
 
     def __init__(
-        self, label, style="default", is_required=False, validators=None, value=None, destination_path=None, options_key=None, options_values=None, **kwargs
+        self,
+        label,
+        style="default",
+        is_required=False,
```

Comment:
@unirho Why is this needed can't this be done with validators like everything else?